package.json

```diff
@@ -17,13 +17,16 @@
     "@types/node": "20.6.3",
     "@types/react": "18.2.22",
     "@types/react-dom": "18.2.7",
+    "country-state-city": "^3.2.1",
     "dotenv": "^16.3.1",
     "eslint-config-next": "13.5.2",
     "eslint-import-resolver-typescript": "^3.6.1",
+    "iso-639-3": "^3.0.1",
```

Comment:
i wonder if this package is TOO comprehensive...🤠 there are just so many variants that typing in something seemingly simple like "English" or "French" gives you _so many_ options

Reply:
lowk maybe 639-1 is better

Reply:
it doesn't look like iso 639-1 support sign languages though, i'll look for an alternative

Reply:
ISO 639-2 provides a good comprehensive list that's not too overwhelming, but unfortunately also doesn't have sign languages. Maybe we can keep a table in Supabase to keep a list of languages that we want? Likely stretch

Reply:
For now, I have it as ISO 639-2 plus languages that end with "Sign Language"

Reply:
ok wonderful! tbh,,, this also sounds like a lot lol

question: thoughts on removing `package.json` changes from this PR and just merging in the big data dropdowns? we could handle the actual package addition in the basic info card sprint since we're not 100% sure which ones we'll be using yet...but also we can also go back and change so no biggie to keep these

Reply:
maybe ISO 639-1 plus sign languages??

Reply:
for `package.json` - could drop after testing, but also not sure if there are better libraries that provide a good list of languages out there. I think we should keep it if we can't find anything else for now so *ideally* we don't have to edit package.json later.